src/Task_21/PersonService.java

```diff
@@ -0,0 +1,94 @@
+package Task_21;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class PersonService {
+
+    public static Set<Person> people = new TreeSet<>(new FirstComparator());
```

Comment:
Like I've sad on the lesson, we shouldn't have class level variables without any need.
Instead of this we SHOULD add return value and incomming parameters to each method to make these methods independent